azure-pipelines.yml

```diff
@@ -14,25 +14,24 @@ pool:
   vmImage: $(imageName)
 
 steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '12.x'
+    displayName: 'Install Node.js'
 
-- task: NodeTool@0
-  inputs:
-    versionSpec: '12.x'
-  displayName: 'Install Node.js'
+  - bash: |
+      /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+      echo ">>> Started xvfb"
+    displayName: Start xvfb
```

Comment:
is this getting interpreted as part of the step?
![image](https://user-images.githubusercontent.com/21194782/72225440-75195500-357d-11ea-8db5-2cec39edcffd.png)

Reply:
CI passed, again I cannot see the pipeline getting "Build not found"

One of the minor rendering issue that lingers on GitHub.

Reply:
This is what it looks like locally:
![image](https://user-images.githubusercontent.com/1661688/72225701-c1b65d80-3588-11ea-9937-84ed6eff248b.png)

Reply:
> CI passed, again I cannot see the pipeline getting "Build not found"

If you login with GitHub you should be able to see it, / maybe log out and log back in.

Reply:
Also the output should be on the check:
https://github.com/jenkinsci/jcasc-vscode-extension/pull/24/checks

Reply:
No dice.
I see no reason to use azure pipeline exclusively?
The guide linked even suggest you simply use vsce npm package to deploy. Nothing prevents using GitHub action than.
https://code.visualstudio.com/api/working-with-extensions/continuous-integration#automated-publishing